articles/spring-cloud/how-to-access-data-plane-azure-ad-rbac.md

```diff
@@ -16,34 +16,48 @@ This article explains how to access the Spring Cloud Config Server and Spring Cl
 ## Assign role to Azure AD user/group, MSI, or service principal
 
-Assign the [azure-spring-cloud-data-reader](../role-based-access-control/built-in-roles.md#azure-spring-cloud-data-reader) role to the [user | group | service-principal | managed-identity] at [management-group | subscription | resource-group | resource] scope.
+Assign the role to the [user | group | service-principal | managed-identity] at [management-group | subscription | resource-group | resource] scope.
+| Role name | Description |
+| - | - |
+| Azure Spring Cloud Config Server Reader | Allow read access to Azure Spring Cloud Config Server |
+| Azure Spring Cloud Config Server Contributor | Allow read, write and delete access to Azure Spring Cloud Config Server |
+| Azure Spring Cloud Service Registry Reader | Allow read access to Azure Spring Cloud Service Registry |
+| Azure Spring Cloud Service Registry Contributor | Allow read, write and delete access to Azure Spring Cloud Service Registry |
 
 For detailed steps, see [Assign Azure roles using the Azure portal](../role-based-access-control/role-assignments-portal.md).
 
 ## Access Config Server and Service Registry Endpoints
 
-After the Azure Spring Cloud Data Reader role is assigned, customers can access the Spring Cloud Config Server and the Spring Cloud Service Registry endpoints. Use the following procedures:
+After the role is assigned, customers can access the Spring Cloud Config Server and the Spring Cloud Service Registry endpoints. Use the following procedures:
 
-1. Get an access token. After an Azure AD user is assigned the Azure Spring Cloud Data Reader role, customers can use the following commands to log in to Azure CLI with user, service principal, or managed identity to get an access token. For details, see [Authenticate Azure CLI](/cli/azure/authenticate-azure-cli). 
+1. Get an access token. After an Azure AD user is assigned the role, customers can use the following commands to log in to Azure CLI with user, service principal, or managed identity to get an access token. For details, see [Authenticate Azure CLI](/cli/azure/authenticate-azure-cli). 
 
     ```azurecli
     az login
     az account get-access-token
     ```
-2. Compose the endpoint. We support default endpoints of the Spring Cloud Config Server and Spring Cloud Service Registry managed by Azure Spring Cloud. For more information, see [Production ready endpoints](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-endpoints). Customers can also get a full list of supported endpoints of the Spring Cloud Config Server and Spring Cloud Service Registry managed by Azure Spring Cloud by accessing endpoints:
+1. Compose the endpoint. We support default endpoints of the Spring Cloud Config Server and Spring Cloud Service Registry managed by Azure Spring Cloud. For more information, see [Production ready endpoints](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-endpoints).
```

Comment:
Is the "Production ready endpoints" hyperlink correct? It doesn't list endpoints of config server or service registry.

Reply:
Updated as we talk offline. Please see the new PR link here. https://github.com/MicrosoftDocs/azure-docs-pr/pull/169498